functions/misc/server.py

```diff
@@ -67,6 +67,13 @@ def get_memory_usage():
         print("error getting memory usage")
         os._exit(2)
 
+def get_ip_address_from_env():
+    try:
+        return(os.environ["HOST_MACHINE_IP"])
```

Comment:
There should be a sane default here, as I don't see a way to programmatically getting the IP of the host from inside the container (but if I'm wrong that would of course work) I think the sane default should be a null value such as `None` or `unset`

The reasoning behind this is not everyone cares about the machine IP as some (myself included)  use the FQDN/hostname to resolve the DNS for the host.